far/FarEng.hlf.m4

```diff
@@ -4225,54 +4225,53 @@ and the '|' character separates include masks from exclude masks.
+cursor down.
 
-    #Gray +# и #Gray -# выбирают и снимают пометку с группы с использованием
-одной или нескольких разделённых запятыми ~масок файлов~@FileMasks@. #Gray *#
-инвертирует текущую пометку. Если параметр #Пометка папок# в диалоге ~Настроек панели~@PanelSettings@
-включен, то пометка меняется и для папок.
+    #Shift+Arrow keys# move the cursor while selecting or deselecting
+items along the way. The action (selection or deselection) depends
+on the state of the item under cursor before pressing the key
+combination.
 
-    #Ctrl+<Gray +># и #Ctrl+<Gray -># выбирают и снимают пометку со всех
-файлов с тем же расширением, что и у файла под курсором. Если параметр
-#Пометка папок# в диалоге ~Настроек панели~@PanelSettings@ включен, то
-пометка меняется и для папок.
+    #Gray +# selects, and #Gray -# deselects the items using one or more
+~File masks~@FileMasks@. #†#
 
-    #Alt+<Gray +># и #Alt+<Gray -># выбирают и снимают пометку со всех файлов
-с тем же именем, что и у файла под курсором. Если параметр #Пометка папок# в диалоге
-~Настроек панели~@PanelSettings@ включен, то пометка меняется и для папок.
+    #Gray *# inverts the current selection. #†#
 
-    #Gray *# инвертирует текущую пометку для файлов, или для файлов и папок,
-если параметр #Пометка папок# в диалоге ~Настроек панели~@PanelSettings@ включен.
+    #Ctrl+<Gray +># selects, and #Ctrl+<Gray -># deselects all items
+with the same #extension# as that of the item under cursor. #†#
 
-    #Ctrl+<Gray *># инвертирует текущую пометку, включая папки.
+    #Alt+<Gray +># selects, and #Alt+<Gray -># deselects all items with
+the same #name# as that of the item under cursor. #†#
 
-    #Shift+<Gray +># и #Shift+<Gray -># выбирают и снимают пометку со всех
-файлов, или для файлов и папок, если параметр #Пометка папок# в диалоге
-~Настроек панели~@PanelSettings@ включен.
+    #Shift+<Gray +># selects, and #Shift+<Gray -># deselects all items. #†#
 
-    #Ctrl+M# (#Восстановить пометку#) восстанавливает предыдущую
-выделенную группу.
+    #†# If the #Select folders# option of the
+~Panel settings~@PanelSettings@ dialog is off, only files are selected
+or deselected. Otherwise, the selection on the folders is changed
+as well.
 
+    #Ctrl+<Gray *># inverts the current selection on all items,
+including folders.
 
- #Пометка мышью#
+    #Ctrl+M# restores the last selection.
 
-    #Щелчок правой кнопкой мыши# помечает или снимает пометку с файла или
-папки под курсором мыши.
+  #Mouse Selection#
 
-    Если щелкнуть правой кнопкой мыши над списком файлов (строка
-с заголовками колонок) или под списком файлов (статусная область) то при
-удерживании правой кнопкой мыши список файлов будет прокручиваться вверх
-или вниз помечая или снимая пометку с файлов, по которым пробегает курсор.
-Действие (пометка или снятие пометки) соответствует состоянию файла под
-курсором до начала прокрутки.
+    #Right click# toggles selection on the clicked item and moves the
+cursor to it.
```

Comment:
`Right click selects files` option in the `Panel settings` does not affect the `Right click` behavior. As per `Settings dialog: panel` help page, if this option is off, right click should open Windows Explorer Context menu. Should I mention the setting here and fix the behavior, or update the `Settings dialog: panel` help page?

Reply:
What exactly do you mean?
It opens a context menu if the option is off and EMenu plugin is loaded.